e2e/etcd_test.go

```diff
@@ -433,9 +433,9 @@ func (ep *etcdProcess) Stop() error {
 }
 
 func (ep *etcdProcess) waitReady() error {
-	readyStrs := []string{"enabled capabilities for version", "published"}
+	readyStrs := []string{"starting server...", "published"}
```

Comment:
Yes, noticed that from the past commits that added this check. But  maybe it is not suitable to check this in waitReady function. This function should only check the server is fully up. But this check will fail the tests in old version of etcd such as 2.3.7. This should not be a must check point in v2 related tests. Maybe it is better to add this check only in v3 related tests?